lib/resources/example_json/r4_examples_encounter.rb

```diff
@@ -1983,6 +1983,13 @@ module Resources
       'serviceProvider': {
         'reference': 'Organization/675844'
       },
+      "location": [
+          {
+            "location": {
+              "reference": "Location/633869"
```

Comment:
can we use  a valid location reference from c1941 in our examples? I think 21250409 is valid.